app/client/views/dashboard/dashboardCtrl.js

```diff
@@ -103,5 +115,64 @@ angular.module('reg')
               });
         });
       };
+      
+      // Ask for location --------------------------------------------------
+      $scope.onCheckIn = function () {
+        if (navigator.geolocation) {
+          navigator.geolocation.getCurrentPosition(
+              ({ coords }) =>  {
+                const { latitude, longitude } = coords;
+                console.log(latitude, longitude);
+                var isCheckedIn = UserService
```

Comment:
Not sure if this will work, since the `makeCheckIn` function is returning a promise